pkg/controller/directvolumemigration/task.go

```diff
@@ -53,8 +53,8 @@ const (
 	DirectVolumeMigrationRsyncCreds         = "directvolumemigration-rsync-creds"
 	DirectVolumeMigrationRsyncTransferSvc   = "directvolumemigration-rsync-transfer-svc"
 	DirectVolumeMigrationRsyncTransferRoute = "dvm"
-	DirectVolumeMigrationStunnelConfig      = "directvolumemigration-stunnel-config"
-	DirectVolumeMigrationStunnelCerts       = "directvolumemigration-stunnel-certs"
+	DirectVolumeMigrationStunnelConfig      = "crane2-stunnel-config"
```

Comment:
do we keep these label values?

Reply:
@alaypatel07 yes, these label values are still passed to Crane lib